notebooks/demo1-join-tables.ipynb

```diff
@@ -0,0 +1,431 @@
+   "metadata": {},
+   "source": [
+    "# Data Manipulation\n",
+    "\n",
+    "The notebook shows how to join two tables on Trino to create a new table. Such a table can then be used for further analysis or for creating visualizations in Apache Superset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "ba19e3b5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pathlib\n",
+    "from dotenv import load_dotenv\n",
+    "import trino\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b36db74d",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Injecting Credentials\n",
+    "\n",
+    "To inject the required access credentials (for S3 and Trino) into this notebook, we will define them as environment variables in a `credentials.env` file, and load them using dotenv. An example of what the contents of `credentials.env` could look like is shown below\n",
+    "\n",
+    "```\n",
+    "# s3 credentials\n",
+    "S3_ENDPOINT=https://s3.us-east-1.amazonaws.com\n",
+    "S3_BUCKET=ocp-odh-os-demo-s3\n",
+    "S3_ACCESS_KEY=xxx\n",
+    "S3_SECRET_KEY=xxx\n",
+    "\n",
+    "# trino credentials\n",
+    "TRINO_USER=xxx\n",
+    "TRINO_PASSWD=xxx\n",
+    "TRINO_HOST=trino-secure-odh-trino.apps.odh-cl1.apps.os-climate.org\n",
+    "TRINO_PORT=443\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "9211c5dd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load credentials\n",
+    "dotenv_dir = os.environ.get(\n",
+    "    \"CREDENTIAL_DOTENV_DIR\", os.environ.get(\"PWD\", \"/opt/app-root/src\")\n",
+    ")\n",
+    "dotenv_path = pathlib.Path(dotenv_dir) / \"credentials.env\"\n",
+    "if os.path.exists(dotenv_path):\n",
+    "    load_dotenv(dotenv_path=dotenv_path, override=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "17c2b5ca",
+   "metadata": {},
+   "source": [
+    "# Join two tables on Trino"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "c44d16fd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create an Trino client\n",
+    "conn = trino.dbapi.connect(\n",
+    "    auth=trino.auth.BasicAuthentication(\n",
+    "        os.environ[\"TRINO_USER\"], os.environ[\"TRINO_PASSWD\"]\n",
+    "    ),\n",
+    "    host=os.environ[\"TRINO_HOST\"],\n",
+    "    port=int(os.environ[\"TRINO_PORT\"]),\n",
+    "    http_scheme=\"https\",\n",
+    "    verify=True,\n",
+    ")\n",
+    "cur = conn.cursor()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "4f146b6d",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[[True]]"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cur.execute(\"drop table default.urgentem.itr_emissions_joined\")\n",
```

Comment:
Is this needed? Why are you starting the operation by dropping a table?

Reply:
+1 will remove it.